app/controllers/weets.js

```diff
@@ -2,6 +2,16 @@ const quotesGardenService = require('../services/quotes_garden');
 const weetService = require('../services/weets');
 const logger = require('../logger/index');
 const { weetMapper } = require('../mappers/weets');
+const { weetObjectSerializer } = require('../serializers/weets');
+
+const getPagingData = (data, limit) => {
```

Comment:
Move this method out of the controller. Maybe a helper would be a good place for this if you made it reusable (with an entity name as param, or without the entity name)

Reply:
You can also serialize the response after this method, in order to make it reusable